pfcp/message/send.go

```diff
@@ -183,138 +149,99 @@ func SendPfcpAssociationSetupRequest(upNodeID pfcpType.NodeID, upfPort uint16) {
 				}
 				pfcpMsgString := string(pfcpMsgBytes)
 				logger.PfcpLog.Debugf("pfcp rsp status ok, %s", pfcpMsgString)
-				pfcpRspMsg := pfcp.Message{}
-				json.Unmarshal(pfcpMsgBytes, &pfcpRspMsg)
+				pfcpRspMsg, err := message.Parse(pfcpMsgBytes)
+				if err != nil {
+					logger.PfcpLog.Errorf("parse pfcp association response failed: %v", err)
+					return err
+				}
 				adapter.HandleAdapterPfcpRsp(pfcpRspMsg, nil)
```

Comment:
Error is not handled here.

Reply:
While I do agree, this was not changed in this PR and I would refrain from making unrelated changes as this PR is large enough. Plus it's worth noting that `HandleAdapterPfcpRsp` never returns anything else than `nil`.

Reply:
OK